src/test/java/com/github/caldav4j/model/request/ContactQueryTest.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright © 2018 Roberto Polli
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.github.caldav4j.model.request;
+
+import org.junit.Test;
+import com.github.caldav4j.BaseTestCase;
+import com.github.caldav4j.util.XMLUtils;
+
+public class ContactQueryTest extends BaseTestCase {
+
+	public ContactQueryTest() {
+		super();
+	}
+    
+    /**
+     *  Don't need BaseTestCase.setUp() here
+     */
+	@Override
+    public void setUp() throws Exception {}
+    
+	@Test
+	public void testPrintContactQuery() {
+		ContactQuery d = new ContactQuery();
+		log.info(XMLUtils.prettyPrint(d)	);
```

Comment:
we could check the actual xml content.

Reply:
Indeed. The approach here is inspired of the existing GenerateQueryTest. It could be a little bit complicated to compare two XML as we may encounter some differences (formatting, etc ...). I didn't think it was worth the effort, but I can do it if necessary.